src/v2/view-builder/internals/BaseView.js

```diff
@@ -77,4 +79,11 @@ export default View.extend({
     return BaseModel.create(currentViewState, optionUiSchemaConfig);
   },
 
+  clearFormError () {
```

Comment:
Why we are overriding this method? Can't the courage handling of clearing error is not sufficient ?

Reply:
This is not override but invoking `clearErrors` on BaseForm.
BaseForm does auto clear error on form submit. For errors from other places (click link at footer) are also being render as form error now hence we have to manually clear it up. (Imaging user click a link multiple times where all requests are error)

Reply:
is it possible to trigger this method only when there are such unbound errors ? Rather then calling this method on every show error.

Reply:
How so. could you elaborate?

BTW, the if condition here is to make this function as no-operation when there is not existing error.

Reply:
Calling this function on line 121, give me impression that it clears all form errors. But it clears only pre-existing errors which can't be automatically cleared.

Also, i was in the impression that showFormErrors only gets called on form submit.

Can you attach a video so that i can understand better?

Reply:
Can you add a video that shows this scenario ?

Reply:
Just wanted to see if there is a way we can avoid doing this.

Reply:
- what's different when you say `all form errors` v.s `pre-existing errors` ?
- `showFormErrors` is a general error handler now
- you can reproduce the issue easily if you can start playground on this topic branch. (remove this call at ` line 121` and click forgot password twice). It might be easier than a video.

Reply:
Ideally Courage Base Form can handle `model.trigger('clearError')` event.
We could add this event to `v2/BaseForm` but I figure calling function is easier to understand entire flow.

Reply:
If we already do `model.trigger('error')`, we can trigger `clearError` as well. I am insisting on this because in one of my PR you suggested to not create functions that does the similar thing in courage. So i wanted to stick to that philosophy, because it made sense :) if `model.trigger('clearError') can do the trick then we should settle for that

Reply:
Sounds good. thanks for catching my inconsistency!